web/src/components/NavigationBarScrollContainer.tsx

```diff
@@ -59,8 +60,26 @@ const getScrollableWidth = (scrollContainer: RefObject<HTMLDivElement>): number
   return 0
 }
 
-const NavigationBarScrollContainer = ({ children, direction, scrollContainerRef }: PropsType): ReactElement => {
-  const [scrollPosition, setScrollPosition] = useState(0)
+const NavigationBarScrollContainer = ({
+  children,
+  direction,
+  scrollContainerRef,
+  activeIndex,
+}: PropsType): ReactElement => {
+  const getInitScrollPosition = useCallback((): number => {
```

Comment:
```suggestion
  const getInitialScrollPosition = useCallback((): number => {
```

Reply:
Why does this have to be a callback? It's only used once during initial render.

Reply:
forgot to readjust since i used it in a useEffect hook before. thx